native/src/main/java/io/ballerina/stdlib/http/api/logging/HttpLogManager.java

```diff
@@ -161,10 +168,22 @@ public void setHttpAccessLogHandler(BMap accessLogConfig) {
             }
         }
 
+        BString logFormat = accessLogConfig.getStringValue(HTTP_LOG_FORMAT);
+        if (logFormat != null &&
+                !(logFormat.getValue().equals(HTTP_LOG_FORMAT_JSON) ||
+                        logFormat.getValue().equals(HTTP_LOG_FORMAT_FLAT))) {
+            stdErr.println("WARNING: Unsupported log format '" + logFormat.getValue() +
+                    "'. Defaulting to 'flat' format.");
+        }
+
+        BArray logAttributes = accessLogConfig.getArrayValue(HTTP_LOG_ATTRIBUTES);
+        if (logAttributes != null && logAttributes.getLength() == 0) {
+            accessLogsEnabled = false;
+        }
+
         if (accessLogsEnabled) {
             System.setProperty(HTTP_ACCESS_LOG_ENABLED, "true");
```

Comment:
When we set a system property, it affects the whole jvm right? If there are 2 services where 1 service has accesslog enabled and other one doesn't, this will confuse the behaviour right?

Reply:
Ack